src/mobile/sgp.mobile.js

```diff
@@ -37,6 +37,7 @@ var localizations = {
   'ru':    ['Мастер-пароль', 'Домена / URL', 'Подтвердить'],
   'nl':    ['Hoofd wachtwoord', 'Domein / URL', 'Genereer'],
   'fy':    ['Haad wachtwurd', 'Domein / URL', 'Ferwurkje']
```

Comment:
need trailing comma on previous line

```suggestion
  'fy':    ['Haad wachtwurd', 'Domein / URL', 'Ferwurkje'],
```